hpx/lcos/local/dataflow.hpp

```diff
@@ -81,13 +81,13 @@ namespace hpx { namespace lcos { namespace local
         template <typename Policy, typename Func, typename Futures>
         struct dataflow_frame //-V690
           : hpx::lcos::detail::future_data<
-                typename hpx::util::invoke_fused_result_of<
+                typename hpx::util::detail::fused_result_of<
```

Comment:
Why did fused_result_of move into the detail namespace while invoke_fused still resides in hpx::util?